lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -189,8 +195,10 @@ def configure(conf)
         # Check for specialized GCE environments (Managed VM or Dataflow).
         # TODO: Add config options for these to allow for running outside GCE?
         attributes = fetch_gce_metadata('instance/attributes/').split
-        if attributes.include?('gae_backend_name') &&
-           attributes.include?('gae_backend_version')
+        if @force_compute_service
```

Comment:
detect_subservice SGTM, I think that makes sense.  Normally we set the service name based on the platform, but when this is true we try to determine if we're really running as part of some other service.